src/main/Extensions/ApplicationSearch/Linux/LinuxApplicationRepository.ts

```diff
@@ -60,7 +60,7 @@ export class LinuxApplicationRepository implements ApplicationRepository {
             (await this.fileSystemUtility.readFile(filePath)).toString(),
         )["Desktop Entry"];
         // https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys
-        const desktopEnv = this.environmentVariableProvider.get("XDG_CURRENT_DESKTOP")?.split(":");
+        const desktopEnv = this.environmentVariableProvider.get("ORIGINAL_XDG_CURRENT_DESKTOP")?.split(":");
```

Comment:
Can we not just use the `LinuxDesktopEnvironmentResolver` here?

Reply:
Some Linux desktop contain other names besides just their desktop (ie. Ubuntu using "ubuntu:GNOME") and even though the desktop environment is just GNOME, we should still check "ubuntu" against the desktop file to determine whether or not we show it